docs/architecture.rst

```diff
@@ -3,132 +3,37 @@
 Architecture
 ============
 
-The Inmanta orchestrator consists of several components:
-
 .. image:: _static/component.*
-   :width: 90%
+   :width: 100%
    :alt: Overview of the Inmanta platform
 
-* The Inmanta **server**: This server manages the deployment process, it keeps track of all agents and the current state of all
-  projects. The server stores it state in PostgreSQL. All other state can be recovered after a server restart or failover.
+* The Inmanta **server**: This server manages the deployment process, it keeps track of all ongoing work.
```

Comment:
Some item in this itemization are put in bold, other items are not. Why is that?